packages/pay/src/controllers/max-packet.ts

```diff
@@ -83,19 +83,18 @@ export class MaxPacketAmountController implements StreamController {
           .subtract(this.verifiedPathCapacity)
           .divideCeil(Int.TWO)
           .add(this.verifiedPathCapacity) as PositiveInt
-
-      case MaxPacketState.UnknownMax:
```

Comment:
If no F08 was encountered after all the initial test packets (state=`UnknownMax`) and then the rate probe tries sending a max u64 packet, that can result in weird behavior -- the exchange rate may get screwed up since it can't be greater than 1:1, for example.